index.js

```diff
@@ -23,13 +23,37 @@ function revDel(options, cb) {
 		var oldManifest = getManifest(options.oldManifest, options.suppress);
 		var newManifest = getManifest(options.newManifest);
 		var oldFiles = getChanged(oldManifest, newManifest);
-
+		
 		if (options.base) {
 			oldFiles = _.map(oldFiles, function (file) {
 				return path.join(options.dest || options.base, file);
 			});
 		}
-
+		if(options.deleteMapExtensions){
+			oldFiles.forEach(function(oldFile){
+				var mapPathCheck = oldFile+".map";
+				if (fs.existsSync(mapPathCheck)) {
```

Comment:
Is there a way to do this without using `exists`?

Also, where possible, would you be able to use the coding style the rest of the code is using (single quotes, whitespace, etc).

Thanks for the PR!